src/librustc/mir/repr.rs

```diff
@@ -824,6 +862,24 @@ impl<'tcx> Lvalue<'tcx> {
             elem: elem,
         }))
     }
+
+    pub fn from_local(mir: &Mir<'tcx>, local: Local) -> Lvalue<'tcx> {
```

Comment:
I still thing we should combine args/vars/temps into one array (locals), but I haven't gotten around to playing with that....